system_test/aest_sync_SUITE.erl

```diff
@@ -384,8 +390,35 @@ net_split_recovery(Cfg) ->
 
     ok.
 
+sync_speed(Cfg) ->
+    InitialNodes = [node_name(I) || I <- lists:seq(1, 4)],
+    NextNodes = [node_name(I) || I <- lists:seq(5, 6)],
+    Height = 1000,
+
+    NodeSpecs = [#{
+        name => N,
+        backend => aest_docker,
+        source => {pull, "aeternity/epoch:v0.11.1"},
+        mine_rate => 100
+    } || N <- InitialNodes ++ NextNodes],
+
+    setup_nodes(NodeSpecs, Cfg),
+    [start_node(N, Cfg) || N <- InitialNodes],
+    wait_for_value({height, Height}, InitialNodes, ?MINING_TIMEOUT * Height, Cfg),
+
+    start_node(node_name(5), Cfg),
+    wait_for_value({height, 0}, [node_name(5)], ?STARTUP_TIMEOUT, Cfg),
+    wait_for_value({height, Height}, [node_name(5)], ?SYNC_TIMEOUT * Height, Cfg),
```

Comment:
This node is not syncing but mining in isolation, right?

Reply:
@lucafavatella Addressed in c575ff2